Formatter/CollectionTwigFormatter.php

```diff
@@ -0,0 +1,92 @@
+<?php
```

Comment:
@rugbymauri Please move these additional formatters to a separate PR, they are unrelated to Symfony 5.